src/api/app/services/workflows/yaml_download_service.rb

```diff
@@ -0,0 +1,33 @@
+module Workflows
+  class YAMLDownloadService
```

Comment:
Just a tiny naming improvement:

```suggestion
  class YAMLDownloader
```

Reply:
Done